packages/frontend-shared/src/locales/en-US.json

```diff
@@ -394,9 +394,9 @@
         "name": "Interactive Run Events",
         "description": "Allows listening to the [`before:run`](https://on.cypress.io/before-run), [`after:run`](https://on.cypress.io/after-run), [`before:spec`](https://on.cypress.io/before-spec), and [`after:spec`](https://on.cypress.io/after-spec) events in plugins during interactive mode."
       },
-      "experimentalSessionSupport": {
-        "name": "Session",
-        "description": "Enable use of [`cy.session()`](https://on.cypress.io/session) and new behavior to handle caching and restoring cookies, localStorage, and sessionStorage."
+      "experimentalSessionAndOrigin": {
```

Comment:
original in [cypress.schema.json](https://github.com/cypress-io/cypress/pull/18075/files#diff-b739e652dfd0f9ab32ffd678b88e9f07c60a1d9ce385f7ae9eaaa4b00764170cR259)